sycl/include/syclcompat/math.hpp

```diff
@@ -47,6 +47,32 @@ namespace complex_namespace = sycl::ext::oneapi::experimental;
 template <typename ValueT>
 using complex_type = detail::complex_namespace::complex<ValueT>;
 
+template <typename ValueT>
+inline ValueT clamp(ValueT val, ValueT min_val, ValueT max_val) {
+  return sycl::clamp(val, min_val, max_val);
+}
+
+#ifdef SYCL_EXT_ONEAPI_BFLOAT16_MATH_FUNCTIONS
+template <>
+inline sycl::ext::oneapi::bfloat16 clamp(sycl::ext::oneapi::bfloat16 val,
```

Comment:
Why not add these directly to the existing bfloat16 extension? So users don't need to mix and max namespaces?
https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/experimental/sycl_ext_oneapi_bfloat16_math_functions.asciidoc

Reply:
That is a good point. We don't have the proper implementation for the backends though. We can set up a TODO to remove this from here, and ask for the proper implementations in the extensions.

Reply:
I've added a TODO regarding this. If SYCL users need this type of `clamp`, they should not go through the syclcompat version. If it gets added to the bfloat extension, it is sensible to remove this from syclcompat.